RetailCoder.VBE/UI/RubberduckUI.de.resx

```diff
@@ -1582,13 +1582,25 @@ Allen Sternguckern, Likern &amp; Followern, für das warme Kribbeln
   <data name="GeneralSettings_ShowLogFolder" xml:space="preserve">
     <value>Logging-Ordner anzeigen</value>
   </data>
-  <data name="information">
+  <data name="information" xml:space="preserve">
     <value />
   </data>
-  <data name="CodeExplorer_CopyToolTip">
+  <data name="CodeExplorer_CopyToolTip" xml:space="preserve">
     <value>In Zwischenablage kopieren</value>
   </data>
-  <data name="cross_circle">
+  <data name="cross_circle" xml:space="preserve">
     <value />
   </data>
+  <data name="SourceControl_PublishRepo_FailureTitle" xml:space="preserve">
+    <value>Veröffentliche Repository</value>
+  </data>
+  <data name="SourceControl_PublishRepoButtonToolTip" xml:space="preserve">
+    <value>Veröffentliche Repository auf Github</value>
+  </data>
+  <data name="SourceControl_PublishRepo_NoOpenRepo" xml:space="preserve">
+    <value>Kein geöffnetes Repository verfügbar, um es auf den Remote-Pfad zu veröffentlichen</value>
+  </data>
+  <data name="SourceControl_ExternalModifications" xml:space="preserve">
+    <value>Eine Quelldatei wurde außerhalb des VBA-Editors geändert. Soll Rubberduck diese nachladen?</value>
```

Comment:
ggf. klarer: "Soll Rubberduck diese Änderungen übernehmen?" alternativ vielleicht auch: "Soll Rubberduck diese Änderungen in den VBA-Editor übernehmen?", wenn man ganz penibel sein wöllte :+1: